### PR TITLE
Re-exporting Falcon-Server components/utils

### PR DESCRIPTION
**Pull request by Borales**
### Please check if the PR fulfills these requirements ###

- [x] The commit message follows our guidelines
- [x] Changelog have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Improvement (to be able to re-use internal utils and components in Falcon-Server based apps).

### CODE REVIEW

File: packages/falcon-server/src/index.ts [hunk 1]
```diff
@@ -30,15 +30,24 @@ import get from 'lodash/get';
...
+import { cacheInvalidatorMiddleware } from './middlewares';
+import { GraphQLCacheDirective, GraphQLCacheInvalidatorDirective } from './schemaDirectives';
 import { Config, BackendConfig } from './types';
 
+export * from './containers';
+export * from './graphqlUtils';
+export * from './middlewares';
+export * from './resolvers';
+export * from './schemaDirectives';
+export * from './utils';
```
Comment (tutok) [inline packages/falcon-server/src/index.ts]: I am not sure if we should export everything from `falcon-server` (which as I knew) should be considered as a tool not like a library. I think things used internally via `falcon-server` should not be exposed, If we need some its internal I think the right way is to move it into `falcon-server-env` (which is as I know, is the set of prerequisites for `falcon-server`)

**Comment by Borales** [comment]
@tutok @nimobeeren `falcon-server/graphqlUtils` module has been moved to `falcon-server-env` for reusability

File: packages/falcon-server/src/index.ts (hunk 1 above)
Comment (Borales) [reply packages/falcon-server/src/index.ts]: done

Comment (tutok) [reply packages/falcon-server/src/index.ts]: so should not we remove this line?

Comment (Borales) [reply packages/falcon-server/src/index.ts]: removed

**Pull request merged**